docs/dictionary/control_st/repeat.lcdoc

```diff
@@ -5,6 +5,22 @@ Type: control structure
 Syntax:     repeat <loopForm>
       <statementList>
     end repeat
+    
+Syntax:  repeat {forever | while <condition> | until <condition>} 
+      <statementList> ; end repeat
```

Comment:
@seaniepie using `;` could be a point of confusion for people so it might be better to move it to its own line. It might be better to remove the original `loopForm` syntax here and delete the associated parameter. Perhaps @livecodeali and or @asayd might want to chime in on that though.